src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -15,6 +15,7 @@ import {
   fetchConfirmedAppointmentDetails,
 } from '../../redux/actions';
 import { getConfirmedAppointmentDetailsInfo } from '../../redux/selectors';
+import { selectFeatureVAOSServiceVAAppointments } from '../../../redux/selectors';
```

Comment:
No need to import `selectFeatureVAOSServiceVAAppointments` since you can retrieve toggle state from `getConfirmedAppointmentDetailsInfo` which is already imported on line 17.

To set up getConfirmedAppointmentDetailsInfo to retrieve toggle state:
Step 1. Add `usev2`  to the deconstructed object constant for useSelector(state) in this current file. See example below to replace lines 25-33

```
  const {
    appointment,
    appointmentDetailsStatus,
    cancelInfo,
    facilityData,
    **useV2**
  } = useSelector(
    state => getConfirmedAppointmentDetailsInfo(state, id),
    shallowEqual,
  );
```

Reply:
@ptouchton ,  I couldn't highlight portion of the code that is collapsed to make suggestion. So I simply indicated by line numbers. Let me know if this make sense to you.